src/main/java/duke/command/DeleteCommand.java

```diff
@@ -0,0 +1,50 @@
+package duke.command;
+
+import duke.Storage;
+import duke.TaskMaster;
+import duke.Ui;
+import duke.exception.DukeException;
+import duke.task.Task;
+
+/**
+ *
+ */
```

Comment:
Might want to add a description for class DeleteCommand. Empty comment seems out of place.

Reply:
Will add it in